src/components/map/RouteMap.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react";
+import { RoutePoiButton } from "./RoutePoiButton";
+import "./RouteMap.scss";
+import "./RoutePoiButton.scss";
+
+interface PoiLocation {
+  x: number;
+  y: number;
+  sol: number;
+}
+
+interface RouteMapProps {
+  roverName: string;
+  mapImageUrl: string;
+  pois: PoiLocation[];
+}
+
+export const RouteMap: React.FunctionComponent<RouteMapProps> = ({
+  roverName,
+  mapImageUrl,
+  pois,
+}) => {
+  const [sol, setSol] = useState<number>();
+  const [photoArr, setPhotosArr] = useState<string[]>([]);
+  const [isLoading, setLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    const url = `https://api.nasa.gov/mars-photos/api/v1/rovers/${roverName}/photos?sol=${sol}&camera=fhaz&api_key=5j3Pvrt0HVQ1PXgmycftrPgdMFA9g0z4NxEBe3dd`;
+
+    const fetchData = async () => {
+      try {
+        setLoading(true)
+        const response = await fetch(url);
+        const json = await response.json();
+        console.log(json.photos);
+        setPhotosArr(json.photos);
+        setLoading(false);
+      } catch (error) {
+        console.log("error", error);
+      }
+    };
+    fetchData();
+}, [sol]);
```

Comment:
This needs to be indented
```suggestion
  }, [sol]);
```